src/oscillator.js

```diff
@@ -626,4 +626,5 @@ p5.SqrOsc = function (freq) {
 
 p5.SqrOsc.prototype = Object.create(p5.Oscillator.prototype);
 
-export const Oscillator = p5.Oscillator;
+const Oscillator = p5.Oscillator;
+export default Oscillator;
```

Comment:
is changing from a named export (that can be imported like `import { Oscillator } from ...`) to a default export (that is imported like `import OscillatorOrWhateverWeWantToCallIt from ...`) going to cause any issues?

Reply:
@therewasaguy i dont think so, its just for this pr., when i will be changing oscillator file i will make it simply 
`export default oscillator`

We can bear it for now 😉

Reply:
ah, ok! I did some more digging and it looks like you're right, this won't cause any issues just yet.

The way you've imported Oscillator in MonoSynth looks good https://github.com/processing/p5.js-sound/pull/509/files#diff-559e243463d4dac174c5eee32f469a81R4

For a future PR, iit looks like Pulse should be importing the module as well in order to extend the class.

https://github.com/processing/p5.js-sound/blob/d30c43d9c688f147a5c2ca678da5fd4f1fe24817/src/pulse.js#L49